### PR TITLE
 Add release schedule for 1.26 release

### PR DESCRIPTION
**Pull request by krol3**
This PR adds 1.26 to the releases schedule for the 1.26.0 release. Following instruction [here](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/docs/Release-Timeline.md#update-releases-page-the-week-before-the-release)
/milestone 1.26
/cc @reylejano @divya-mohan0209 @onlydole

### CODE REVIEW

**Comment by sftim** [comment]
If this gets /lgtm and /approve, a tech lead (or SIG chair) can manually trigger a GitHub merge even whilst the block is in effect.

**Comment by sftim** [comment]
The build error is
```
3:02:09 PM: Error: Error building site: "/opt/build/repo/content/zh-cn/releases/patch-releases.md:168:1": failed to render shortcode "release-branches": failed to process shortcode: "/opt/build/repo/layouts/shortcodes/release-branches.html:16:17": execute of template failed: template: shortcodes/release-branches.html:16:17: executing "shortcodes/release-branches.html" at <time.Format>: error calling Format: unable to cast <nil> of type <nil> to Time
```

File: content/en/releases/patch-releases.md [hunk 1]
```diff
@@ -84,7 +84,15 @@ releases may also occur in between these.
...
-{{< release-branches >}}
+### 1.26
+
+Next patch release is **1.26.1**
+
+End of Life for **1.26** is **2024-10-27**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE | NOTE |
+|---------------|----------------------|-------------|------|
+| 1.25.1        | 2022-12-06           | 2022-12-08  |      |
```
Comment (sftim) [inline content/en/releases/patch-releases.md:95]: I'd expect to see
```suggestion
{{< release-branches >}}
```
here

File: content/en/releases/patch-releases.md [hunk 2]
```diff
@@ -84,6 +84,16 @@ releases may also occur in between these.
...
+### 1.26
+
+Next patch release is **1.26.1**
+
+End of Life for **1.26** is **2024-10-27**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE | NOTE |
+|---------------|----------------------|-------------|------|
+| 1.26.1        | 2022-12-06           | 2022-12-08  |      |
+
```
Comment (sftim) [inline content/en/releases/patch-releases.md:96]:
```suggestion
```
See https://deploy-preview-38340--kubernetes-io-vnext-staging.netlify.app/releases/patch-releases/#1-26

File: data/releases/schedule.yaml [hunk 3]
```diff
@@ -1,4 +1,12 @@
 schedules:
+- release: 1.26
+  releaseDate: 2022-12-08
```
Comment (onlydole) [inline data/releases/schedule.yaml:3]:
```suggestion
  releaseDate: 2022-12-09
```

**Comment by nate-double-u** [comment]
/approve
Thanks for the update!

**Pull request merged**